src/data/map.hpp

```diff
@@ -28,8 +28,311 @@
 
 namespace rigel { namespace data {
 
-using ActorID = std::uint16_t;
+enum class ActorID
```

Comment:
This enum class is a mixture of Actors, FX, etc.
From my point of view a separation doesn't make much sense, but that's to be decided by @lethal-guitar

Reply:
Was thinking about it - I have some ideas on how it could be separated, but for now,  let's leave it like this :+1: